WalletWasabi.Tests/RegressionTests/CoinJoinTests.cs

```diff
@@ -297,7 +301,8 @@ public async Task CcjTestsAsync()
 				await aliceClient.PostUnConfirmationAsync();
 			}
 
-			round = coordinator.GetCurrentInputRegisterableRoundOrDefault();
+			round = coordinator.GetCurrentInputRegisterableRoundOrDefault()!;
```

Comment:
@lontivero Would it make sense to you to switch this to" `bool TryGetCurrentInputRegisterableRound([NotNullWhen(true)] out CoordinatorRound? coordinatorRound)` too?

Reply:
I did that. We can revert the last commit if it is not useful.